sharktank/requirements.txt

```diff
@@ -1,7 +1,12 @@
+iree-turbine
+
 # Runtime deps.
 gguf==0.6.0
 numpy==1.26.3
```

Comment:
I ran into errors trying to unpin these versions (replacing `==` with `>=`). Sample logs: https://github.com/nod-ai/SHARK-Platform/actions/runs/11714432277/job/32629101709#step:6:64

Maybe one of the deps doesn't support numpy 2.0? I can play dependency-updates-whack-a-mole to try unpinning _some_ of the packages I guess...?

Reply:
We probably need to, yes :/ Can help with that on Monday. Should probably fill a follow up issue to keep track.